Sources/Sentry/SentryUIEventTracker.m

```diff
@@ -46,21 +46,27 @@ - (void)start
                     return;
                 }
 
+                NSString *operation = @"ui.action";
+                if (event
+                    && (event.type == UIEventTypePresses || event.type == UIEventTypeTouches)) {
```

Comment:
m: 'UIEventTypePresses' is related with press of a physical button. I don't think this counts as a click.
Everything related to the screen is considered an `UIEventTypeTouches`, for example, UISlider valueChanged and UITextField editingChanged, and IMO those should not be considered as a click.

Reply:
Thanks, will update the logic.